packages/react-components/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -26,26 +26,31 @@ const useRootStyles = makeStyles({
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
     ...shorthands.borderWidth(0),
     cursor: 'pointer',
-    display: 'flex',
-    flexDirection: 'row',
+    display: 'grid',
     flexShrink: 0,
```

Comment:
I guess this should be removed as it has no effect with gap ?

Reply:
This would only affect its flex behavior within a `display: flex` parent, so changing its own `display` value shouldn't have an effect.

Reply:
Tabs are within a flex parent and I need them not to shrink to support react-overflow.